workspaces/announcements/plugins/announcements-backend/src/service/announcementsContextBuilder.ts

```diff
@@ -42,6 +43,7 @@ export type AnnouncementsContext = {
   permissionsRegistry: PermissionsRegistryService;
   persistenceContext: PersistenceContext;
   signals?: SignalsService;
+  notificationSvc: NotificationService;
```

Comment:
I understand the intent behind appending `Svc`, similar to how it could be `notificationsClient`. Because we already follow the pattern of `SignalService -> signals`, `AnnouncementsService -> announcements`, `PermissionsService -> permissions`, can we get this updated to be `NotificationService -> notifications`?